src/cmd/serve.rs

```diff
@@ -241,35 +240,38 @@ pub fn serve(
     let (tx, rx) = channel();
     let mut watcher = watcher(tx, Duration::from_secs(1)).unwrap();
     watcher
-        .watch("content/", RecursiveMode::Recursive)
+        .watch(root_dir.join("content/"), RecursiveMode::Recursive)
         .map_err(|e| ZolaError::chain("Can't watch the `content` folder. Does it exist?", e))?;
     watcher
-        .watch(config_file, RecursiveMode::Recursive)
+        .watch(root_dir.join(config_file), RecursiveMode::Recursive)
         .map_err(|e| ZolaError::chain("Can't watch the `config` file. Does it exist?", e))?;
 
-    if Path::new("static").exists() {
+    let mut dir = root_dir.join("static");
+    if dir.exists() {
         watching_static = true;
         watcher
-            .watch("static/", RecursiveMode::Recursive)
+            .watch(dir, RecursiveMode::Recursive)
             .map_err(|e| ZolaError::chain("Can't watch the `static` folder.", e))?;
     }
 
-    if Path::new("templates").exists() {
+    dir = root_dir.join("templates");
+    if dir.exists() {
         watching_templates = true;
         watcher
-            .watch("templates/", RecursiveMode::Recursive)
+            .watch(dir, RecursiveMode::Recursive)
             .map_err(|e| ZolaError::chain("Can't watch the `templates` folder.", e))?;
     }
 
-    if Path::new("themes").exists() {
+    dir = root_dir.join("themes");
+    if dir.exists() {
```

Comment:
can we name them properly instead of reusing `dir`? It's easy to miss an assignment